portal/src/graphql/portal/AuthenticationLoginIDSettings.tsx

```diff
@@ -130,10 +131,11 @@ function setLoginIdKeyEnabled(loginIdKey: LoginIDKeyConfig, enabled: boolean) {
 }
 
 function constructAppConfigFromState(
-  appConfig: PortalAPIAppConfig,
+  rawAppConfig: PortalAPIAppConfig,
+  effectiveAppConfig: PortalAPIAppConfig,
```

Comment:
`effectiveAppConfig` seems unused in this function. Remove it if possible.

Reply:
removed in later commit